src/timessquare/storage/nbhtmlcache.py

```diff
@@ -0,0 +1,139 @@
+"""The Redis-based cache of rendered HTML pages."""
+
+from __future__ import annotations
+
+from typing import Any, AsyncIterable, Mapping, Optional
+
+import aioredis
+
+from timessquare.domain.nbhtml import NbHtmlModel
+
+from .redisutils import calculate_key
```

Comment:
Nit: Mixed relative and absolute imports.

Reply:
To be honest I'm finding it easier to maintain code with only relative imports within the subpackage, and absolute imports otherwise so I'll stick with this.